### PR TITLE
KHADRI-1-simple-save-operations

### PR DESCRIPTION
**Pull request by vikram-8790**

### CODE REVIEW

File: src/com/khadri/hibernate/client/ClientMain.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import org.hibernate.Transaction;
+import org.hibernate.cfg.Configuration;
+
+import com.khadri.hibernate.entity.Employee;
+
+
+public class ClientMain {
+
+	public static void main(String[] args) {
+		Configuration cfg = new Configuration();
```
Comment (khadri2021) [inline src/com/khadri/hibernate/client/ClientMain.java]: Separate code changes of session creation to the file BasicOperationSession.java
give package name as com.khadri.hibernate.session

**Review by khadri2021 — COMMENTED** [review]
Do changes as per review comments.

File: src/com/khadri/hibernate/client/ClientMain.java (hunk 1 above)
Comment (vikram-8790) [reply src/com/khadri/hibernate/client/ClientMain.java]: completed the changes

File: src/com/khadri/hibernate/client/ClientMain.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package com.khadri.hibernate.client;
+
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+import com.khadri.hibernate.entity.Employee;
+import com.khadri.hibernate.session.BasicOperationSessipon;
+
+public class ClientMain {
```
Comment (khadri2021) [inline src/com/khadri/hibernate/client/ClientMain.java]: change file name from ClientMain to Client1

File: src/com/khadri/hibernate/client/ClientMain.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+	public static void main(String[] args) {
+
+		Session sess = BasicOperationSessipon.getSession();
+
+		Transaction txn = sess.beginTransaction();
+
+		Employee emp = new Employee();
+		emp.setEmpName("JHON");
```
Comment (khadri2021) [inline src/com/khadri/hibernate/client/ClientMain.java]: emp name and salary read from the scanner

File: src/com/khadri/hibernate/entity/Employee.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
+package com.khadri.hibernate.entity;
+
+public class Employee {
+
+	private Integer empId;
+	private String empName;
+	private Double salary;
```
Comment (khadri2021) [inline src/com/khadri/hibernate/entity/Employee.java]: change from salary to empSalary

File: src/com/khadri/hibernate/session/BasicOperationSessipon.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
+
+package com.khadri.hibernate.session;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class BasicOperationSessipon {
```
Comment (khadri2021) [inline src/com/khadri/hibernate/session/BasicOperationSessipon.java]: correct typo mistake from BasicOperationSessipon to BasicOperationSession

**Review by khadri2021 — COMMENTED** [review]
Add folder sql and tables.sql for your database scripts.

File: src/com/khadri/hibernate/client/ClientMain.java (hunk 2 above)
Comment (vikram-8790) [reply src/com/khadri/hibernate/client/ClientMain.java]: completed

**Review by khadri2021 — APPROVED** [review]

**Pull request merged**